### PR TITLE
fix: add disabled state to tabs

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/2651
#### Please provide a brief summary of this pull request.
There is added disabled state for tabs. It's not shown in examples, but property is opened to be changed by passing `[disabled]="true"`

image
tabs

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by JKMarkowski** [comment]
@InnaAtanasova I can't reproduce it, could you check again? Maybe it was some netlify cache

File: libs/core/src/lib/tabs/tab-list.component.ts [hunk 1]
```diff
@@ -116,8 +117,8 @@ export class TabListComponent implements AfterViewInit, OnChanges, OnDestroy {
     }
 
     /** @hidden */
-    tabHeaderClickHandler(tabIndex: number): void {
-        if (this.selectedIndex !== tabIndex) {
+    tabHeaderClickHandler(tabIndex: number, disabled?: boolean): void {
```
Comment (salarenko) [inline libs/core/src/lib/tabs/tab-list.component.ts:120]: Wouldn't it be better to be consistent here and use `this._canBeSelected(tabIndex)` as well?

Comment (JKMarkowski) [reply libs/core/src/lib/tabs/tab-list.component.ts]: I think we can't have example of disabled in docs, because it's not designed by fiori3

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-07-06 at 11 44 50 AM

It's still reproducible, just apply `is-disabled` class to the `li` element, then click on the first tab, press the `tab` key to go to the next Tab, it gets the focus and it shouldn't, when you press Enter it selects the disabled Tab

**Pull request merged**